ingest-app/src/cmr/ingest/services/ingest_service.clj

```diff
@@ -5,6 +5,7 @@
     [cmr.ingest.services.ingest-service.service]
     [cmr.ingest.services.ingest-service.util]
     [cmr.ingest.services.ingest-service.variable]
+    [cmr.ingest.services.ingest-service.subscription]
```

Comment:
imported namespaces should be sorted in alphabetic order.

Reply:
Done.